AdvLoggerPkg/UnitTests/AdvancedLoggerWrapper/AdvancedLoggerWrapperTestApp.c

```diff
@@ -185,6 +185,7 @@ InitializeInMemoryLog (
 
     if (!ValidateInfoBlock ()) {
       mLoggerInfo = NULL;
+      UT_ASSERT_TRUE (FALSE);

```

Comment:
nit: 
```suggestion
      UT_ASSERT_NOT_NULL (mLoggerInfo);

```

Just for when reading the log